pkg/jobs/scheduled_job_executor.go

```diff
@@ -77,7 +77,7 @@ type ScheduledJobController interface {
 		schedule *ScheduledJob,
 		txn *kv.Txn,
 		descsCol *descs.Collection,
-	) error
+	) (int, error)
```

Comment:
you should document what int result means.

Reply:
I'd make the comment more clear, as "how many rows" is quite vague. What about:

"OnDrop runs before the passed in `schedule` is dropped as a part of a `DROP SCHEDULE` query. OnDrop may drop the schedule's dependent schedules and will return the number of additional schedules it drops".